### PR TITLE
chore: clean up JetBrains resource requirements

### PR DESCRIPTION
**Pull request by ericpaulsen**
removes explicit requirements around workspaces using JetBrains IDEs. also fixes the header in `Custom Image Creation`. this was missed in #811.

### CODE REVIEW

File: images/writing.md [hunk 1]
```diff
@@ -80,26 +80,29 @@ USER root
...
+# Add a binary to the PATH that points to the IDE startup script.
+RUN ln -s /opt/[IDE]/bin/idea.sh /usr/bin/[IDE]
 ```
 
 Make sure that you replace `[IDE]` with the name of the IDE in lowercase and
 provide its
 [corresponding `[CODE]`](https://plugins.jetbrains.com/docs/marketplace/product-codes.html).
 
-More specifically, here's how to install the CLion IDE onto your image:
+More specifically, here's how to install the IntelliJ IDE onto your image:
```
Comment (johnstcn) [inline images/writing.md:93]:
```suggestion
More specifically, here's how to install IntelliJ IDEA Community Edition into your image:
```

File: images/writing.md [hunk 2]
```diff
@@ -80,26 +80,29 @@ USER root
...
-RUN curl -L \
-"https://download.jetbrains.com/product?code=CL&latest&distribution=linux" \
-| tar -C /opt/clion --strip-components 1 -xzvf
-RUN ln -s /opt/clion/bin/clion.sh /usr/bin/clion
+# Install intellij
+RUN mkdir -p /opt/idea
+RUN curl -L "https://download.jetbrains.com/product?code=IIC&latest&distribution=linux" \
+| tar -C /opt/idea --strip-components 1 -xzvf -
+
+# Add a binary to the PATH that points to the Intellij startup script.
```
Comment (johnstcn) [inline images/writing.md:106]:
```suggestion
# Create a symbolic link in PATH that points to the Intellij startup script.
```

File: workspaces/editors.md [hunk 3]
```diff
@@ -149,15 +149,6 @@ The symlink names supported by Coder are:
 - `studio` (Android Studio)
 - `webstorm`
 
-### System requirements
-
-We recommend running the JetBrains IDE in an workspace with the following
-minimum resources but the resources required depends on your workspace-specific
-requirements:
```
Comment (johnstcn) [inline workspaces/editors.md:156]: Should we direct users to consult the documentation for their favoured IDE? 
For Idea, we could link directly to https://www.jetbrains.com/help/idea/installation-guide.html#requirements

**Review by johnstcn — APPROVED** [review]

File: workspaces/editors.md (hunk 3 above)
Comment (ericpaulsen) [reply workspaces/editors.md:156]: How would we do this efficiently without linking to each IDE's specific requirements?

Comment (khorne3) [reply workspaces/editors.md:156]: I don't think there's a great way of linking to all the pages, so we could just say something like

> Please refer to the system requirements documentation for your IDE to determine the adequate amount of resources for your workspace

**Review by khorne3 — APPROVED** [review]

**Pull request merged**